### DISCUSSION TITLE
[WARNING PDN-0195] Removing 5 via(s) between M2 and M5 at (35.7900 um, 1291.9420 um) for VSS

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#5158] [General]
What does it mean and what should I do?

From latest https://github.com/The-OpenROAD-Project/megaboom PDN

**Comment by maliberty** [comment]
@gadfort any comments?

**Reply by gadfort** [reply]
it's just letting them know that some vias were not inserted. They would need to go check the area to ensure it's not going to cause an issue with IR or EM.

**Reply by oharboe** [reply]
How are these fixed?

**Reply by gadfort** [reply]
Without seeing the issue, I cannot predict the reason for why these were not inserted. Generally it's because of macro halos, or metal width issues that ultimately prevent vias from getting inserted.
It's also possible you these missing vias will ultimately not present an issue for the design.

**Comment by oharboe** [comment]
I see... Not an unfamiliar situation in EDA tools.... 

In C++ one can generally remove all warnings, not so in FPGA/EDA tools....

I going to forego filing a github issue on this case for now.
